2-Operating-Systems/1-arch.md

```diff
@@ -0,0 +1,459 @@
+# Install Arch Linux: The TL;DR
+
+## Overview
+
+This guide is meant to be an opinionated and to the point guide on installing
+Arch Linux. Power users with different opinions should refer to the official
+[Arch Linux installation guide][1].
+
+[1]:https://wiki.archlinux.org/index.php/Installation_guide
+
+## Requirements
+
+* These steps are best done on a Linux or OSX computer to run early steps.
+  * Windows users may use WSL (Windows Subsystem for Linux)
+  * Or can use an [Ubuntu live CD][1]
+* A blank flash drive, 1GB+
+* Target computer
+  * Known to be compatible with Linux
+  * Supports UEFI boot
+  * Has wireless card
+  * Does not require proprietary "out of tree" drivers
+* Time: 1-3 hours, depending on optional steps
+
+[1]: https://www.ubuntu.com/download/desktop
+
+## Device Support
+
+Devices this guide is known to work with:
+    * Dell XPS 13 or 15 series
+    * Lenovo Yoga series
+    * GPD Pocket series
+
+## Goals
+
+* Simple partitioning scheme
+* Full disk encyption
+* Passwordless: Use GPG Smartcard for login, sudo, ssh
+
+## Steps
+
+1. Download latest Arch Linux ISO image file
+   1. Install torrent client such as aria2
+   2. Copy latest .iso.torrent URL address from [Arch Linux releases][1]
+   3. Download ISO with aria2.
+      
+      Example:
+      
+      ```
+      aria2c --on-download-complete=exit https://YOUR_TORRENT_URL_HERE
+      ```
+
+[1]: https://mirrors.kernel.org/archlinux/iso/latest/
+
+2. Write latest ISO to flash drive
+   
+   1. List current storage devices with `lsblk`
+   2. Insert flash drive
+   3. List current storage devices with `lsblk` again
+   4. Take note of new drive. Example: /dev/sdX
+   5. Use `dd` to write ISO image to drive
+      
+      ```
+      sudo dd \
+      bs=4M \
+      if=archlinux-DATE-x86_64.iso \
+      of=/dev/sdX \
+      status=progress \
+      oflag=sync
+      ```
+   6. Remove flash drive
+
+3. Boot flash drive of target laptop
+   
+   1. Insert flash drive
+   2. Boot laptop to BIOS screen
+      * e.g. F1, F2, F10, Delete, Escape while computer starts booting
+   3. Check that 'secure boot' is Disabled
+   4. Ensure USB drives will boot by default
+   5. Reboot into Arch Linux installation environment
+
+4. If needed: Set desired keyboard layout (if not US standard)
+   
+   Example: German
+   
+   ```
+   loadkeys de-latin1
+   ```
+
+5. If needed: Change rotation   
```

Comment:
Isn't obvious: rotation of what?

Also I've never had an issue with wrong rotation before. Is this something you've experienced?